service/src/main/frontend/src/pages/DataDictionary.vue

```diff
@@ -0,0 +1,295 @@
+                size="9px"
+                color="cyan-8"
+                round
+                dense
+                @click="
+                  {
+                    props.expand = !props.expand;
+                    Formatters.toggleVisibility(props.row);
+                  }
+                "
+                :icon="props.row.isVisible.value ? 'remove' : 'add'"
+                v-if="Formatters.buttonParse(props.cols, props.row)"
+              />
+              <q-icon
+                  style="margin-left: 4px;"
+                  size="1rem"
+                  :name="'bi-arrow-right'"
+                  color="cyan-8"
+                  v-if="props.row.duplicate == 1"
+              />
+            </q-td>
+            <q-td
+              v-for="col in props.cols"
+              :key="col.name"
+              :props="props"
+              style="font-size: 13px;"
+              :title="Formatters.parseVal(col.name, col.value)"
+              @click="Feature.copyLabel(col.value)"
+            >
+              <label style="cursor: pointer;">
+                {{
+                Formatters.maxSubstring(
+                  Formatters.parseVal(col.name, col.value), col.name
+                )
+              }}
+              </label>
+            </q-td>
+          </q-tr>
+        </template>
+      </q-table>
+    </div>
+  </main>
+  <div v-if="banner?.enabled" :style="banner?.styleTop" style="margin-top: 0.50vh;">
+      {{ banner?.messageTop }}
+  </div>
+  <div v-if="banner?.enabled" :style="banner?.styleBottom">
+      {{ banner?.messageBottom }}
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { QTable, QTableProps, exportFile, useQuasar, Notify } from 'quasar';
+import { useToggle, useDark } from '@vueuse/core';
+import { api } from '../boot/axios';
+import { Banner, columns } from '../functions/components';
+import * as Formatters from '../functions/formatters';
+import * as Wrapper from '../functions/csvWrapper';
+import * as Feature from '../functions/features';
+
+// Defines the Table References, loading for axios, search filter, and pagination to sort.
+const $q = useQuasar();
+const table = ref();
+const loading = ref(true);
+const filter = ref('');
+const changeFilter = ref('');
+const banner = ref<Banner>();
+let rows: QTableProps['rows'] = [];
+let paginationFront = ref({
+  rowsPerPage: 200,
+  sortBy: 'fieldName',
+});
+
+// API - Defines all the Nececssary API calls for the user, and filters.
+// Note that to run the endpoint in DEV mode, you must build the project at least once first.
+onMounted(() => {
+  let endpointData = '';
+  let bannerData = 'banner';
+  if (process.env.DEV) {
+    endpointData = 'data/v2/'
+    bannerData = 'data/v2/banner/'
+  }
+
+  api
+  .get(bannerData, undefined)
+  .then((response) => {
+    banner.value = response.data as Banner;
+  })
+  .catch((reason) => {
+    console.log('Could not fetch banner: ' + reason);
```

Comment:
Probably more of an error here.
```suggestion
    console.error('Could not fetch banner: ' + reason);
```